### ISSUE TITLE
Helm repo add rancher-stable failed

### ISSUE THREAD

**Issue by dommgifer** [rancher/rancher#22902]
I try to use helm to install rancher.

But use command
`helm repo add rancher-stable https://releases.rancher.com/server-charts/stable`,
got error:
`Error: Looks like "https://releases.rancher.com/server-charts/stable" is not a valid chart repository or cannot be reached: Failed to fetch https://releases.rancher.com/server-charts/stable/index.yaml : 403 Forbidden`

But test url used command
`curl https://releases.rancher.com/server-charts/stable/index.yaml`,
it was worked.

How can I solved it?

**Comment by superseb** [comment]
What version of Helm are you using? (`helm version --client`)

Sounds like a (transparent) proxy interfering and returning the 403, and curl possibly has a proxy configured?

**Comment by dommgifer** [comment]
I tested v2.12.1 and v2.14.3 version.

**Comment by superseb** [comment]
Without additional info, there is nothing much to say about this. Are you in a network restricted environment? What proxy configuration is set in the shell environment where you are executing the command and what has curl configured? What does `curl -v` show which network path is being used?

Closing it as not a bug, I added `rancher-stable` repository today and yesterday using v2.14.3 without problems but if you provide more info I can look into it.
